soak_bench_scripts/soak.sh

```diff
@@ -1,7 +1,8 @@
-#!/bin/bash
+#!/usr/local/bin/bash
```

Comment:
Just curious - do we need this change, and if so, why?

Reply:
Yes, this is related to the bash version on MacOS - it is very out of date and I had to install a newer version. Security controls on the Mac prevent me *replacing the default version unless I jump through some hoops to do so - this was my preferred approach.